docs/providers/aws/guide/variables.md

```diff
@@ -51,8 +51,6 @@ You can define your own variable syntax (regex) if it conflicts with CloudFormat
 - [Read String Variable Values as Boolean Values](#read-string-variable-values-as-boolean-values)
 - [Pseudo Parameters Reference](#aws-cloudformation-pseudo-parameters-and-intrinsic-functions)
 
-## Casting string variables to boolean values
-
```

Comment:
Empty section, probably a typo from this PR: https://github.com/serverless/serverless/pull/6869/files#diff-e9c53a4188d707906258b6d8c262e5cf84d4314fd48d26fe32b5171637d5cdd8R51